### DISCUSSION TITLE
Random cat

### DISCUSSION THREAD

**Discussion by Arubinu** [Aqueuse/NekoV2#4] [Ideas]
It might be nice to **generate a random** cat.
I'm not an expert, but it would be cool to offer **several races** (black/white, ginger, etc ...) in a list, putting the **default choice in random** for the first launch.

**Comment by Aqueuse** [comment]
I could do a parameter window with a choice of assets, yes ! I'm not very good for drawing pixel art cats on the other hand, this would be the major challenge :p
